python/sglang/srt/layers/activation.py

```diff
@@ -81,17 +81,20 @@ def forward_npu(self, x: torch.Tensor) -> torch.Tensor:
         out = torch_npu.npu_swiglu(x)
         return out
 
+    def forward_xpu(self, x: torch.Tensor) -> torch.Tensor:
+        d = x.shape[-1] // 2
+        output_shape = x.shape[:-1] + (d,)
+        out = torch.empty(output_shape, dtype=x.dtype, device=x.device)
+        silu_and_mul(x, out)
+        return out
```

Comment:
call `forward_native` from forward_xpu util we have C++ kernels ready.

we don't have to rewrite it again here.